support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStepExecution.java

```diff
@@ -78,6 +83,29 @@ public boolean start() throws Exception {
         }
     }
 
+    private List<ParameterValue> completeDefaultParameters(List<ParameterValue> parameters, Job<?,?> project) {
+        List<ParameterValue> completeListOfParameters = Lists.newArrayList(parameters);
+        List<String> names = Lists.transform(parameters, new Function<ParameterValue, String>() {
+            @Override public String apply(ParameterValue input) {
+                return input.getName();
+            }
+        });
+        if (project != null) {
```

Comment:
:ant: we already did a null check in line 48, so this is unnecessary (and the parameter could be `@Nonnull`).